test/lit/passes/optimize-added-constants-memory64.wast

```diff
@@ -0,0 +1,56 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
+
+;; RUN: wasm-opt %s -all --roundtrip --optimize-added-constants --low-memory-unused -S -o - | filecheck %s
+
+(module
+ (memory $0 i64 1 4294967296)
+
+;; CHECK:       (func $load_i64 (result i64)
+;; CHECK-NEXT:   (i64.load
+;; CHECK-NEXT:   (i64.const 579)
+;; CHECK-NEXT:  )
+;; CHECK-NEXT: )
+;; CHECK-NEXT: (func $load_overflow_i64 (result i64)
+;; CHECK-NEXT:  (i64.load offset=32
+;; CHECK-NEXT:   (i64.const -16)
+;; CHECK-NEXT:  )
+;; CHECK-NEXT: )
+;; CHECK-NEXT: (func $store
+;; CHECK-NEXT:  (i64.store
+;; CHECK-NEXT:   (i64.const 579)
+;; CHECK-NEXT:   (i64.const 123)
+;; CHECK-NEXT:  )
+;; CHECK-NEXT: )
+;; CHECK-NEXT: (func $store_overflow
+;; CHECK-NEXT:  (i64.store offset=32
+;; CHECK-NEXT:   (i64.const -16)
+;; CHECK-NEXT:   (i64.const 123)
+;; CHECK-NEXT:  )
+;; CHECK-NEXT: )
```

Comment:
@tlively , these automatically-generated checks don't look right - they aren't right before each function etc. Any idea?

Reply:
The functions looks like they should be handled ok. @dongAxis, did you actually run `./scripts/auto_update_tests.py lit`? If not, it would be good to run that and push the fixed checks so we don't get unnecessary churn in later PRs.

Reply:
address the comment, thanks